lib/Dropdown/DropdownTrigger.js

```diff
@@ -16,27 +17,44 @@ class DropdownTrigger extends Component {
   };
 
   render() {
-    const { children, useButton, ...rest } = this.props;
-    return useButton ? (
-      <div
-        ref={trigger => {
-          this.trigger = trigger;
-        }}
-        className="dropdown__trigger-wrapper"
-      >
-        <Button {...this.props} clickHandler={this.handleClick}>
-          {children}
-        </Button>
-      </div>
-    ) : (
+    const { children, useButton, useSelect, ...rest } = this.props;
+    const { showContent } = this.context[GATHER_UI_DROPDOWN];
+    const createRef = trigger => {
```

Comment:
Is there any difference here to using `React.createRef()`?

Reply:
Aye I believe so. `React.createRef` creates an object which includes additional ref properties in it like `this.ref.current`. I just want to pass a callback which assigns the ref to the instance I am working with 👍

It was by accident that they were called the same and maybe the end up being the same thing but I prefer callbacks personally.